AstronomCore/src/main/java/com/astronomvm/core/model/data/type/DataType.java

```diff
@@ -10,6 +10,7 @@
     INPUT_FLOW_NAME,
     FUNCTIONAl_MODEL,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. ([squid:S00115](https://sonarcloud.io/organizations/ahmedriahi-github/rules#rule_key=squid%3AS00115))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=AhmedRiahi_AstronomVM&pullRequest=22&issues=AWZfQr2kDZywvPe4Lful&open=AWZfQr2kDZywvPe4Lful)

[ISSUE_KEY]: AWZfQr2kDZywvPe4Lful